okhttp/src/main/java/com/squareup/okhttp/Dispatcher.java

```diff
@@ -177,4 +177,12 @@ synchronized void executed(Call call) {
   synchronized void finished(Call call) {
     if (!executedCalls.remove(call)) throw new AssertionError("Call wasn't in-flight!");
   }
+
+  public int getRunningCallCount() {
+    return runningCalls.size();
+  }
+
+  public int getQueuedCallCount() {
```

Comment:
Added.

Also, I have an irrational fear that metrics systems may call this too many times and then interfere with actual request processing. That's irrational right?